vm/src/builtins/float.rs

```diff
@@ -472,6 +479,26 @@ impl PyFloat {
             })
     }
 
+    #[cfg(feature = "num-bigint")]
+    #[pymethod]
+    fn as_integer_ratio(&self, vm: &VirtualMachine) -> PyResult<(PyIntRef, PyIntRef)> {
```

Comment:
what's happened for this function? was there incompatibility?

Reply:
Yes, took this code from the reverted PR.
https://github.com/RustPython/RustPython/pull/4952/files#diff-d030ca0d2309f322c97032ab7179319e19b3806317cac7194ce0d0ef6c55ff1aL461